cmd/minikube/cmd/start.go

```diff
@@ -229,7 +234,15 @@ func runStart(cmd *cobra.Command, args []string) {
 		glog.Errorln("Error saving profile cluster configuration: ", err)
 	}
 
+	if shouldCacheImages {
+		fmt.Println("Waiting for image caching to complete...")
+		if err := groupCacheImages.Wait(); err != nil {
+			glog.Errorln("Error caching images: ", err)
```

Comment:
Should this be a warning instead of an error?

Reply:
Hi. I would suggest that it should be an error: if I asked it to cache images and it failed then that is an error which I would like to investigate as lack of the cached images in subsequent calls to "start" could be end up with the daemon downloading the images without any indication that it is doing that. Also, if there is a problem with caching and an error is returned I can easily disable the caching. With a warning it's too easy to miss this all of this happening.

Reply:
Ah, right. SGTM.